tools/error_report/error_report.py

```diff
@@ -0,0 +1,343 @@
+import sys
```

Comment:
Please add Samsung copyright and license sentence

Reply:
Ok I'll add. I was confused because it is python.